index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Porfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--MAIN CONTENT-->
+    <main class="main">
```

Comment:
Great design @borrnelsonk 🤝

- [ ] [Optional] Kindly add the background image to your design as it appears in the Figma template.
![Background](https://user-images.githubusercontent.com/63917725/186647955-15d97d4c-74fa-456a-88fe-4e6cd0e8de6b.PNG)

Reply:
- [ ] Kindly follow the your chosen Figma design as close as possible. As mentioned by the previous reviewers, kindly add the background image to your design as it appears in the Figma template you chose.

![figma-background-image](https://user-images.githubusercontent.com/26597432/186680008-7dbaa003-5041-45c2-bc5a-67e2f3ce5e7e.png)